templates/full-time-program.html

```diff
@@ -18,8 +18,8 @@ <h2>
     <div class="column">
       <h1 class="blue-h1">The Program</h1>
       <p class="justify">
-        Techtonica's full-time software engineering training program is a free,
-        virtual, year-long program that provides the opportunity for
+        Techtonica's full-time software engineering training program is a 
+        virtual, close to year-long program that provides the opportunity for
```

Comment:
“Techtonica’s full-time software engineering training program is a virtual, close to year-long program that provides the opportunity for” —> “Techtonica provides the opportunity for”

Reply:
### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).